app/src/main/res/values/styles.xml

```diff
@@ -7,11 +7,4 @@
         <item name="colorPrimaryDark">@color/colorPrimaryDark</item>
         <item name="colorAccent">@color/colorAccent</item>
     </style>
-
-
-    <style name="MySeekBar" parent="Widget.AppCompat.SeekBar">
```

Comment:
why you remove this?
this is for the sliding

Reply:
It produced a bunch or errors and is not compatible with our current minimum API version